### ISSUE TITLE
Implementation with python 3.9

### ISSUE THREAD

**Issue by s-goldman** [darioflute/sospex#34]
At the moment, for users with the most up-to-date version of python (3.9), the conda installation throws the following error:

```
UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - sospex -> python[version='2.7.*|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.8,<3.9.0a0']

Your python: python=3.9

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.
```

Also for the pip install using python 3.9, I get the following error after running `sospex`:

```
Traceback (most recent call last):
  File "/Users/srgoldma/opt/anaconda3/bin/sospex", line 4, in <module>
    from sospex import start
  File "/Users/srgoldma/opt/anaconda3/lib/python3.9/site-packages/sospex/start.py", line 2, in <module>
    from sospex import mainwindow
  File "/Users/srgoldma/opt/anaconda3/lib/python3.9/site-packages/sospex/mainwindow.py", line 23, in <module>
    from sospex.apertures import photoAperture,PolygonInteractor, EllipseInteractor, RectangleInteractor, PixelInteractor
  File "/Users/srgoldma/opt/anaconda3/lib/python3.9/site-packages/sospex/apertures.py", line 4, in <module>
    from matplotlib.mlab import dist_point_to_segment
ImportError: cannot import name 'dist_point_to_segment' from 'matplotlib.mlab' (/Users/srgoldma/opt/anaconda3/lib/python3.9/site-packages/matplotlib/mlab.py)
```

**Comment by darioflute** [comment]
Anaconda Python 3.9 just become public. I will test sospex against it asap.
Thanks for raising the issue

**Comment by darioflute** [comment]
I did not yet release a compiled version for anaconda Python 3.9
The pip version is very old (version 33, I think) and does not work with the matplotlib distribution.
One of these days, I should update it.
Thanks,
Dario

**Comment by s-goldman** [comment]
@darioflute 
Hey Dario, I installed sospex from the github distribution and it seems to work fine with my setup.

Macbook Pro (M1 chip) running Mojave (12.5.1) and python 3.9. 

At the moment, the sospex pip install doesn't let you install sospex if you are running python 3.9, so maybe you can add somewhere that people can get around that by installing it from the github distribution using `python setup.py install`.

**Comment by darioflute** [comment]
Finally I released a version for Python 3.9 !
Closing the issue.
